tests/test_db.py

```diff
@@ -138,3 +139,83 @@ def teardown_module(module):
 
         assert await db.allocate_parking_lot("test_user", 1) is True
         assert await db.allocate_parking_lot("test_user", 2) is False
+
+
+@pytest.mark.asyncio
+async def test_get_available_parking_lots(event_loop):
```

Comment:
I'm guessing these tests work because of your choice of location coordinates, you might want to give a brief explanation of this? It's not immediately obvious to me who doesn't know the code all that well.

Reply:
This is due to 1 degree of latitude equaling to about 110km and the distance of 1 degree of longitude depends on the exact latitude.

Reply:
@uptonfunk, this reminds me that the simulation team should be aware that we are using earth coordinates at the backend to calculate distances.